ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -359,99 +360,105 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
               isLastItem={true}
             />
 
-            <SectionHeaderComponent
-              sectionHeader={I18n.t("profile.main.developersSectionHeader")}
-            />
-            {
-              // since no experimental features are available we avoid to render this item (see https://www.pivotaltracker.com/story/show/168263994).
-              // It could be useful when new experimental features will be available
-              /*
-              this.developerListItem(
-              I18n.t("profile.main.experimentalFeatures.confirmTitle"),
-              this.props.isExperimentalFeaturesEnabled,
-              this.onExperimentalFeaturesToggle
-            )*/
-            }
-
-            {(this.props.isPagoPATestEnabled ||
-              this.state.showPagoPAtestSwitch) &&
-              this.developerListItem(
-                I18n.t("profile.main.pagoPaEnvironment.pagoPaEnv"),
-                this.props.isPagoPATestEnabled,
-                this.onPagoPAEnvironmentToggle,
-                I18n.t("profile.main.pagoPaEnvironment.pagoPAEnvAlert")
-              )}
-
-            {this.developerListItem(
-              I18n.t("profile.main.debugMode"),
-              this.props.isDebugModeEnabled,
-              this.props.setDebugModeEnabled
-            )}
-
-            {this.props.isDebugModeEnabled && (
-              <React.Fragment>
-                {this.debugListItem(
-                  `${I18n.t("profile.main.appVersion")} ${getAppLongVersion()}`,
-                  () => {
-                    if (this.state.numberOfTaps === 4) {
-                      this.setState({ showPagoPAtestSwitch: true });
-                    } else {
-                      const numberOfTaps = this.state.numberOfTaps + 1;
-                      this.setState({
-                        numberOfTaps
-                      });
-                    }
-                  },
-                  false
-                )}
-
-                {backendInfo &&
-                  this.debugListItem(
-                    `${I18n.t("profile.main.backendVersion")} ${
-                      backendInfo.version
-                    }`,
-                    () => clipboardSetStringWithFeedback(backendInfo.version),
-                    false
-                  )}
-                {sessionToken &&
-                  this.debugListItem(
-                    `Session Token ${sessionToken}`,
-                    () => clipboardSetStringWithFeedback(sessionToken),
-                    false
+            {/* Developers Section */}
+            {isDevEnv && (
+              <>
```

Comment:
We could avoid empty tags by using a wrapper View (`<View></View>`)
It's the same but the _tsx_ keeps well readable